cmd/osv-scanner/callanalysis_parser.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+var stableCallAnalysisStates = map[string]bool{
+	"go":   true,
+	"rust": false,
+}
+
+// Creates a map to record if languages are enabled or disabled for call analysis.
+func createCallAnalysisStates(enabledCallAnalysis []string, disabledCallAnalysis []string) map[string]bool {
+	callAnalysisStates := make(map[string]bool)
+
+	for _, language := range enabledCallAnalysis {
+		callAnalysisStates[language] = true
+	}
+
+	for _, language := range disabledCallAnalysis {
+		callAnalysisStates[language] = false
+	}
+
+	enableAll, containsAll := callAnalysisStates["all"]
+	for language, isStable := range stableCallAnalysisStates {
+		if _, exists := callAnalysisStates[language]; !exists {
+			callAnalysisStates[language] = isStable
+			if enableAll {
+				callAnalysisStates[language] = true
+			}
```

Comment:
you can use `||` here, since really `enableAll` is for enabling non-stable languages:

```suggestion
			callAnalysisStates[language] = isStable || enableAll
```